packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -229,6 +230,15 @@ export function buildRootObject() {
       return zcfMint;
     };
 
+    /** @type SetTestJig */
+    const setTestJig = testFn => {
```

Comment:
```suggestion
    const setTestJig = (testFn = () => {}) => {
```
If we only want to return `zcf` can we default to an empty function?